### DISCUSSION TITLE
Disable OK Button on Utility Window

### DISCUSSION THREAD

**Discussion by JasonJrSmith** [grahampugh/erase-install#381] [Q&A]
Hello Graham, 

Thanks for producing this script and workflow... It works amazing! 

I wanted to know if there's a way to disable the OK button on the utility screen, which displays the progress bar when downloading the OS. As well as when verifying the OS before the authenticated restart. 

If and when a user clicks OK, the dialog window disappears, but the processes are still running in the background, unbeknownst to the user. This leaves them in a 'confused' state. 

So far I've tried adding the `--button1disable` flag to 3 different arrays in the `erase-install.sh`. (see attached photos). Have I done that correctly?

Discussion thread #345 references a similar question - but slightly different from my ask. 

Any help would be greatly appreciated. Thanks in advance. 

<img width="712" alt="Screenshot 2023-03-14 at 10 39 26 PM" src="https://user-images.githubusercontent.com/78504097/225204271-58654979-5c27-40d6-91c4-3aa1f4937408.png">

<img width="538" alt="Screenshot 2023-03-14 at 10 56 33 PM" src="https://user-images.githubusercontent.com/78504097/225203886-acfef372-5b42-4678-bbd5-c3c0d8999466.png">
<img width="487" alt="Screenshot 2023-03-14 at 10 55 46 PM" src="https://user-images.githubusercontent.com/78504097/225203880-c6394f94-86b9-42f5-a35d-0f4b7e58827e.png">
<img width="446" alt="Screenshot 2023-03-14 at 10 56 05 PM" src="https://user-images.githubusercontent.com/78504097/225203885-4806028c-7e66-4b76-b505-ed8a4590ad52.png">

**Comment by grahampugh** [comment]
The point in the OK button is to allow the user to continue to work. If the OK button was disabled, then the user can't work. If you don't want them to work, you can use the --fs option.

**Reply by JasonJrSmith** [reply]
Makes sense... thank you.
